manage/sawtooth_manage/docker.py

```diff
@@ -163,14 +163,28 @@ def start(self, node_config):
         node_num = node_name[len('validator-'):]
         for proc in state['Processors']:
             compose_dict['services'][proc] = {
-                'image': 'sawtooth-{}'.format(proc),
+                'image': 'sawtooth-dev-{}'.format(proc),
                 'expose': ['40000'],
                 'links': ['validator'],
                 'volumes': ['/project:/project'],
                 'container_name': '-'.join([self._prefix, proc, node_num]),
                 'entrypoint': 'bin/{} tcp://{}:40000'.format(proc, node_name)
             }
 
+        # start the rest_api for the first node only
```

Comment:
@nick-drozd The use case didn't call for that. The idea was to be able to query blocks, state etc. to one of the validators in the network. 

@delventhalz Could you or @peterschwarz interject?